ONSdigital/census-rh-ui
# Cr 1005 address changed

# Motivation and Context
Change to Authentication journey, so that if user selects 'No' to confirm address, they are taken through the enter/select/confirm route. Manual edit is no longer available.

# What has changed
Removal of modify_address route (modify functionality is being kept in RHUI, RHSvc and Cucumber, as it my still be needed, but not by this route)
Introduction of 'change address' route, using the existing common enter/select/confirm process, and RHSvc UAC link endpoint
Addition of missing unit tests for direct access throughout start journey
Also includes function call to RHSvc renaming, and traps for censusAddressType being 'NA'

Code requires RHSvc CR-1000 and RHCuc CR-1005 before deploy

# How to test?
Enter UAC, select no to 'Confirm Address', select new address, get confirmation page and launch EQ. Enter UAC again to see newly selected address.
Full unit test coverage

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1005
https://collaborate2.ons.gov.uk/jira/browse/CR-1000
https://github.com/ONSdigital/census-rh-cucumber/pull/69
